test/kubetest/pods/kernel_forwarder.go

```diff
@@ -0,0 +1,116 @@
+//nolint
+package pods
+
+import (
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// KernelDataplanePod creates a pod
+func KernelDataplanePod(name string, node *v1.Node) *v1.Pod {
+	return createKernelDataplanePod(name, node, nil, nil, nil)
+}
+
+// KernelDataplanePodConfig creates a pod with config
+func KernelDataplanePodConfig(name string, node *v1.Node, variables map[string]string) *v1.Pod {
+	return createKernelDataplanePod(name, node, nil, nil, variables)
+}
+
+// KernelDataplanePodLiveCheck creates a pod with live check
+func KernelDataplanePodLiveCheck(name string, node *v1.Node) *v1.Pod {
+	return createKernelDataplanePod(name, node, createProbe("/liveness"), createProbe("/readiness"), nil)
+}
+
+func createKernelDataplanePod(name string, node *v1.Node, liveness, readiness *v1.Probe, variables map[string]string) *v1.Pod {
+	ht := new(v1.HostPathType)
+	*ht = v1.HostPathDirectoryOrCreate
+
+	priv := true
+
+	mode := v1.MountPropagationBidirectional
+	pod := &v1.Pod{
+		ObjectMeta: v12.ObjectMeta{
+			Name: name,
+		},
+		TypeMeta: v12.TypeMeta{
+			Kind: "Deployment",
+		},
+		Spec: v1.PodSpec{
+			HostPID: true,
+			Volumes: []v1.Volume{
+				{
+					Name: "workspace",
+					VolumeSource: v1.VolumeSource{
+						HostPath: &v1.HostPathVolumeSource{
+							Type: ht,
+							Path: "/var/lib/networkservicemesh",
+						},
+					},
+				},
+				{
+					Name: "postmortem",
```

Comment:
I bet you don't need that. AFAIK it was introduced to debug VPP/VPP-agent when the tests fail.

Reply:
Removed it 👍